src/vsphere_cpi/lib/cloud/vsphere/cluster_picker.rb

```diff
@@ -30,7 +30,7 @@ def best_cluster_placement(req_memory:, disk_configurations:)
         datastore_picker.update(accessible_datastores)
 
         begin
-          placement = datastore_picker.best_disk_placement(disk_configurations)
+          placement = datastore_picker.best_disk_placement(disk_configurations, max_swapfile_size=req_memory)
```

Comment:
This doesn't exactly do what I think you're trying to do; specifically it:

1. Declares a new local variable `max_swapfile_size`
2. Assigns `req_memory` to `max_swapfile_size`
3. Passes the result of the assignment as the second argument

Mostly it's the same result but unnecessarily creates a variable. Ruby has no direct equivalent of Python's ability to assign any function argument by name with `=`.

Reply:
Got you! I will just use `req_memory` as a second parameter then